docs/dev/webapp/getting-started.md

```diff
@@ -0,0 +1,73 @@
+# Getting Started
+
+The Web App is a single page application written in TypeScript with Angular 5.
+If you are new to Angular you can check out the [Angular Tour of Heroes](https://angular.io/tutorial) it will help you getting started.
+
+## Development environment setup
+
+### Editor 
+
+There are two editors that can be recommended:
+
+- [JetBrains WebStorm](https://www.jetbrains.com/webstorm/) (non-free) 
+- [Visual Studio Code](https://code.visualstudio.com/) (open source)
+
+Both of them have excellent TypeScript and Angular 5 support.
+
+### Package manager
+
+This project uses the [yarn](https://yarnpkg.com) package manager. It is also possible to use [npm](https://www.npmjs.com/). 
+
+### Angular CLI
+
+The  [Angular CLI](https://cli.angular.io/) is used to serve and build our web app. 
+It also can be used to create components, services and much more.
+Read its [documentation](https://github.com/angular/angular-cli/wiki) to check out how it is installed and what else it is feasible of.
+
+### CORS Plugin
+
+Since the idea of the WebApp is to be deployed with the API on the same server you need a plugin that allows cross origin requests.
+
+- Firefox: [CORS Everywhere](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/?src=search)   
+- Chrome/Chromium: [Allow-Control-Allow-Origin: \*](https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi)
+
+## Get the WebApp running
```

Comment:
Ok, you got a tiny problem.. so far, I've read `WebApp`, `web app` and `Web App`.